config/sync/field.field.citation.su_article_journal.su_journal_publisher.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 65af4483-34fd-4b30-9caa-8c95557c80d7
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.citation.su_journal_publisher
+    - stanford_publication.citation_type.su_article_journal
+id: citation.su_article_journal.su_journal_publisher
+field_name: su_journal_publisher
+entity_type: citation
+bundle: su_article_journal
```

Comment:
All citation fields aren't prefixed with a bundle. it makes it easier to build the citation for the different bundles.
Also I don't see this field getting added to the citation format. Is that intentional?

Reply:
That's correct, the ticket did not request any change to the citation formats, so I left well enough alone.

Reply:
I didn't want to change the machine name for `su_publisher`, to preserve backwards compatibility, so I made it `su_journal_publisher` to differentiate the new field.  It's only used in the Journal citation type.